src/task_types/resmoke_tasks.rs

```diff
@@ -421,18 +404,6 @@ impl GenResmokeTaskServiceImpl {
     ) -> Result<Vec<SubSuite>> {
         let origin_suite = multiversion_name.unwrap_or(&params.suite_name);
         let test_list = self.get_test_list(params, multiversion_name)?;
-        let required_stats_count = percent_of_tests(test_list.len(), REQUIRED_STATS_THRESHOLD);
-        if task_stats.test_map.len() < required_stats_count {
```

Comment:
Since we are distributing tests without historic runtime data after we have processed test with historic data, we don't need this special logic anymore.